InscryptionAPI/Card/CardExtensions.cs

```diff
@@ -1056,6 +1057,43 @@ public static TailParams SetLostTailPortrait(this TailParams info, Sprite portra
 
     #region Helpers
 
+    /// <summary>
+    /// Create a card in a specific slot based off the name. A much more robust way that's the same as `yield return BoardManager.Instance.CreateCardInSlot()`.
+    /// </summary>
+    /// <param name="cardName">The name of the card.</param>
+    /// <param name="slotToSpawnIn">The slot to spawn in.</param>
+    /// <param name="transitionLength">Time to transition the card to the slot. The longer the time, the longer it will take to be placed at the slot.</param>
+    /// <param name="resolveTriggers">Whether or not to activate other card triggers.</param>
+    /// <returns>The enumeration of the card being placed in the slot.</returns>
+    public static IEnumerator CreateCardInSlot(
+        this string cardName,
```

Comment:
god no please no why would we ever have extension methods on string the only thing this does is bring terror and shame